sdk/core/core-client/src/base64.browser.ts

```diff
@@ -40,3 +40,18 @@ export function decodeString(value: string): Uint8Array {
   }
   return arr;
 }
+
+/**
+ * Converts a uint8Array to a string.
+ */
+export function uint8ArrayToString(ab: Uint8Array): string {
+  return decodeURIComponent(
+    escape(
+      Array.from(ab)
+        .map(function(item) {
+          return String.fromCharCode(item);
+        })
+        .join("")
```

Comment:
this can be simplified

```suggestion
      Array.from(ab, item => String.fromCharCode(item)).join("")
```